### ISSUE TITLE
FYI to pubg players

### ISSUE THREAD

**Issue by jayrodathome** [OneMooreByte/RocketChat#4]
Just want to leave this here. It took me 3 windows installs and 4 copies of PUBG from steam to recreate this issue and find out what was going on.

History: I have a steam account https://steamcommunity/id/jayrodathome which is about 11 years old now. Roughly 12k of gaming and level 285 which for me was an acomplishment. 3khrs + on CSGO, 4khrs + on TF2 and 4Khours + on Rocket League.

**My impatience:** So PUBG hit the seen I was about 700 hours deep and at the time about $1,200 invested in items, luckily had sold a bunch earlier like the alpha crates etc. anyway I am horrible at this game and would always die first. So I started to spool up Rocket League and at least practice some air dribbles while my teammates died and we could start fresh. Being an advocate for this program Rocket Chat I had always kept in running on my system but until I started playing Rocket League it wasn't actually doing anything while PUBG was running.

**Day after:** So day one after running rocket league for the first time during death I sign into PUBG and BAM hit with a Game ban, turns into a vac ban. No questions can be asked nothing can be resolved I'm a dirty cheater. I was mind blown and didn't connect the two. I opened support ticket and got no response from blueballs.

Me knowing I wasn't cheating but still wanting to play I went and setup another steam account and bought PUBG ($40) and ofc another copy of RL so I wouldn't get bored ($12). 3 hours of game play.. banned. Now I was getting suspicious. Something was obviously wrong. I opened another suppport claim with blueballs they said "even if you're not hacking in pubg any hack detected on system will ban you. (I have no hacking software of any kind on my system.

Being the dummy I am: So of course what do I do? I buy another copy of PUBG and another Copy of Rocket League (because yeah I'm that guy) but this time I decided I was going to wipe my entire system first before installing and playing so I wouldn't get banned for any stupid file that was obviously being picked up by battle eye. Few Hours later..... Playing pubg, Playing Rocket league Boom banned. At this point I had 3 programs running Rocket Chat, Pubg, Rocket League and Of course Steam and the default Windows programs.

**Research:** I started looking at Rocket chat because it was the only thing that was abnormal and injection of sorts. I found some articles saying that .exe versions of auto hotkey can trigger false positives with battle eye I guess. So when pubg was running and I was playing Rocket League and submitting all the mini hacks to the game while playing it was enough for Battle Eye to see it as a hack.

To no avail: I submitted everything I had found to Battle Eye and to PUBG in excruciating Detail. I ended up buying the game 3 more times and could accurately recreate the ban and or not ban. I thought this is gold. They will maybe thank me for finding a false positive.

**Their Response:** Please stop emailing us we are removing your account. You will never be unbanned.

SO Much for my service and hard work and money! I no longer play PUBG. I do plan to continue to use this program.

**TLDR:** Make sure you exit this program when running pubg just to be safe. Certainly never run it and use it in RL while running pubg at the same time. Cheers!

**Comment by OneMooreByte** [comment]
Hello, sorry for late response, I have not checked Github in a long time.
I am sorry this was an issue. I will add a disclaimer somewhere and hope no one else will have this issue.
